src/App.tsx

```diff
@@ -1,7 +1,39 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string;
+};
+
+export class App extends React.Component<{}, State> {
+  state: Readonly<State> = {
+    pressedKey: '',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.onKeyUpHandler);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.onKeyUpHandler);
+  }
+
+  onKeyUpHandler = (event: KeyboardEvent) => {
+    const formattedKeyCode = event.code
+      .split('Key')
+      .join('');
```

Comment:
Тобі просто потрібно перевірити чи там 'Space'
Твоя імплементація не спрацює

```suggestion
    const formattedKeyCode = event.code === 'Space'
      ? 'Space'
      : event.code
```